index.html

```diff
@@ -1534,7 +1576,7 @@ <h4 class="modal-title">Source Code</h4>
     <script src="dist/js/material.min.js"></script>
     <script src="dist/js/ripples.min.js"></script>
     <script>
-      $.material.init();
+      //$.material.init();
```

Comment:
Their was a problem where `$.material.init()` was being called twice, so strange things were happening, suppose I should have removed the line rather than comment it out.